charts/openproject/templates/seeder-job.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "common.names.fullname" . }}-seeder-{{ now | date "20060102150405" }}
```

Comment:
We put the current time in the job name so we don't need to work with hooks to re-run the seeder on each upgrade.